hydrosdk/deployment_configuration.py

```diff
@@ -0,0 +1,474 @@
+from dataclasses import dataclass, field
+from typing import List, Dict
+
+from hydrosdk.cluster import Cluster
+from hydrosdk.utils import handle_request_error, enable_camel_case
+
+
+@enable_camel_case
+@dataclass
+class HorizontalPodAutoScalerSpec:
+    """
```

Comment:
The HPA specs from Kubernetes allow to configure packets-per-second/requests-per-second scenarios. Will it be easy to extend this implementaiton further to include those cases as well?

Reply:
If I understood you correctly, your comment does not apply directly to the SDK itself. 

This SDK is just a thin wrapper for the `api/v2/deployment_configuration` API. If you have any questions\comments about API design It's better to ask @KineticCookie

Reply:
Yeah, this spans multiple codebases.

Reply:
I guess, if anything will be changed from manager part, we can just add a new field here, like `packets-per-second`, and use its value as a threshold to trigger scale up.